spec/services/creators/other_incomes_creator_spec.rb

```diff
@@ -14,7 +14,14 @@ module Creators
       }
     end
 
-    subject { described_class.call(params) }
```

Comment:
Could `params` defined on line 8 be deleted if it's not being used here? (I can't see anywhere else where it's called but could be missing something)

Reply:
Nice catch... I deleted the second one after replacing it and forgot to delete this one too!